flink-test-utils-parent/flink-test-utils-junit/src/main/java/org/apache/flink/util/TestLogger.java

```diff
@@ -31,7 +31,10 @@
 /**
  * Adds automatic test name logging. Every test which wants to log which test is currently executed
  * and why it failed, simply has to extend this class.
+ *
+ * @deprecated Continue using {@link TestLoggerExtension} with Junit5 tests.
  */
+@Deprecated
```

Comment:
Hi @JingGe , I'm not sure shoule these classes releted to junit4 be marked to `Deprecated`. Would you mind helping check it? thanks a lot~

Reply:
Hi @X-czh , could you remove this commit, and squash all commit after address other comments?

Because most of junit4 classes are not marked to `Deprecated` so far, if you want to do it, it's better to do it in a separate JIRA, and mark all classes togother.

WDYT?